src/main/java/seedu/address/model/person/Person.java

```diff
@@ -116,6 +153,26 @@ public Github getGithub() {
         return github;
     }
 
+    public void markWeekPresent(int week) {
+        this.weeksPresent.add(week);
+    }
+
+    public void unmarkWeekPresent(int week) {
```

Comment:
This block of code seemed to be unused, could perhaps remove it if it is not needed.